sonar-project.properties

```diff
@@ -0,0 +1,36 @@
+#Z = Github Username or organization name (scallait)
+#A = project name (PolicyLock)
+
+sonar.projectKey=scallait_bootcamp-project-2021
+sonar.projectName=bootcamp-project-2021
+
+# =====================================================
+#   Meta-data for the project
+# =====================================================
+
+sonar.links.homepage=https://github.com/Z/A
+sonar.links.ci=https://travis-ci.org/Z/A
+sonar.links.scm=https://github.com/Z/A
+sonar.links.issue=https://github.com/Z/A/issues
+
+
+# =====================================================
+#   Properties that will be shared amongst all modules
+# =====================================================
+
+sonar.host.url=https://sonarcloud.io
+sonar.organization=scallait-github
+sonar.login=6a653c2fc24a7541293b505ab91cc31fbb04742e
```

Comment:
idk if these are varaibles I should have access too haha. I do really like how you added sonarcloud to your site tho, as it is a great resouce and it might be used in the nonprofit projects down the road. This file probably doesn't need to be commited tho, or you could hide the important information using ENV variables [https://docs.github.com/en/actions/learn-github-actions/environment-variables](url)